API/Controllers/AuthenticationController.cs

```diff
@@ -43,5 +49,12 @@ public async Task<IActionResult> LogoutAsync([FromBody] UserLogoutDTO userLogout
             await _authenticationService.LogoutAsync(userLogoutDTO);
             return Ok();
         }
+
+        [HttpPost("confirm-email")]
+        public async Task<IActionResult> ConfirmEmail(EmailConfirmationRequestDTO request)
```

Comment:
I suggest looking for a user not by email, but by id from the access token, which will be in the request header. So it will be much more securely